setup.py

```diff
@@ -218,10 +218,16 @@ def run(self):
     extras_require={
         'R': find_requirements("requirements-extras-r.txt"),
         'Prophet': find_requirements("requirements-extras-prophet.txt"),
+        'shell': [],
+        'sagemaker-shell': [],
     },
     entry_points=dict(
         console_scripts=[
-            "gluonts-validate-dataset=gluonts.dataset.validate:run"
+            "gluonts-validate-dataset=gluonts.dataset.validate:run",
+            "gluonts-train=gluonts.shell.train:main [shell]",
+            "gluonts-serve=gluonts.shell.serve:main [shell]",
```

Comment:
Will remove these lines.

The shell is quite SageMaker specific and thus I’m not sure we need them.